src/ui/marker.js

```diff
@@ -31,6 +34,9 @@ type Options = {
  * @param {PointLike} [options.offset] The offset in pixels as a {@link PointLike} object to apply relative to the element's center. Negatives indicate left and up.
  * @param {string} [options.color='#3FB1CE'] The color to use for the default marker if options.element is not provided. The default is light blue.
  * @param {boolean} [options.draggable=false] A boolean indicating whether or not a marker is able to be dragged to a new position on the map.
+ * @param {number} [options.rotatation=0] The rotation angle of the marker (in degrees), relative to it's respective {@link Marker#rotationAlignment} setting.
```

Comment:
suggestion: "(clockwise, in degrees)"

Reply:
nit: `its` - no apostrophe? (https://theoatmeal.com/comics/apostrophe)

Reply:
Both resolved